src/site/content/en/media/video-and-source-tags/index.md

```diff
@@ -0,0 +1,341 @@
+it can play the provided video file. If it can't, the enclosed text displays.
+
+```html
+<video src="chrome.webm" type="video/webm">
+    <p>Your browser cannot play the provided video file.</p>
+</video>
+```
+
+### Specify multiple file formats
+
+Recall from [File basics](../file-basics) that not all browsers support the same
+video formats. The `<source>` element lets you specify multiple formats as a
+fallback in case the user's browser doesn't support one of them.
+
+For example the code below produces the ebedded video that immediately follows.
+
+```html
+<video controls>
+  <source src="https://storage.googleapis.com/web-dev-assets/video-and-source-tags/chrome.webm" type="video/webm">
+  <source src="https://storage.googleapis.com/web-dev-assets/video-and-source-tags/chrome.mp4" type="video/mp4">
+    <p>Your browser cannot play the provided video file.</p>
+</video>
+```
+
+[Try it](https://googlesamples.github.io/web-fundamentals/fundamentals/media/video-main.html)
+
+When the browser parses the `<source>` tags, it uses the optional `type`
+attribute to determine which file to download and play. If the browser
+supports WebM, it plays that; if not, it checks whether it can play
+MPEG-4 videos.
+
+This approach has several advantages over serving different HTML or server-side
+scripting, especially on mobile:
+
+* You can list formats in order of preference.
+* Native client-side switching reduces latency; only one request is made to
+  get content.
+* Letting the browser choose a format is simpler, quicker, and potentially
+  more reliable than using a server-side support database with user-agent detection.
+* Specifying each file source's type improves network performance; the browser can select a
+  video source without having to download part of the video to "sniff" the format.
+
+These issues are especially important in mobile contexts, where bandwidth and
+latency are at a premium and the user's patience is likely limited. Omitting the
+type attribute can affect performance when there are multiple sources with
+unsupported types.
+
+{% Aside %} There are a few ways you can dig into the details. Check out [A
+Digital Media Primer for Geeks](//www.xiph.org/video/vid1.shtml) to find out
+more about how video and audio work on the web. You can also use [remote
+debugging](https://developers.google.com/web/tools/chrome-devtools/remote-debugging)
+in DevTools to compare network activity [with type
+attributes](https://googlesamples.github.io/web-fundamentals/fundamentals/media/video-main.html)
+and [without type
+attributes](https://googlesamples.github.io/web-fundamentals/fundamentals/design-and-ux/responsive/notype.html).
+
+Also check the response headers in your browser developer tools to
+[ensure your server reports the right MIME type](//developer.mozilla.org/en/docs/Properly_Configuring_Server_MIME_Types);
+otherwise video source type checks won't work.
+{% endAside %}
+
+### Specify start and end times
```

Comment:
@rkochman You had an earlier comment about range request material not making much sense. (Ignore my suggestion that it be removed.) I found some additional information about fragments, for which range requests are a prerequisite. I've improved the section  and improved how the concepts are tied to each other. Please read this whole section again.

Reply:
I do like that--thanks!